spec/models/reports/facility_appointment_scheduled_days_spec.rb

```diff
@@ -24,19 +24,23 @@
 
   it "buckets and counts appointments by the number of days between creation date and scheduled date" do
     facility = create(:facility)
-    patients = create_list(:patient, 4, recorded_at: 1.month.ago)
-    _appointment_scheduled_0_to_14_days = create(:appointment, patient: patients.first, facility: facility, scheduled_date: 10.days.from_now, device_created_at: Time.current)
-    _appointment_scheduled_15_to_30_days = create(:appointment, patient: patients.second, facility: facility, scheduled_date: 16.days.from_now, device_created_at: Time.current)
-    _appointment_scheduled_31_to_60_days = create(:appointment, patient: patients.third, facility: facility, scheduled_date: 36.days.from_now, device_created_at: Time.current)
-    _appointment_scheduled_more_than_60_days = create(:appointment, patient: patients.fourth, facility: facility, scheduled_date: 70.days.from_now, device_created_at: Time.current)
+    create(:appointment, scheduled_date: -1.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
+    create(:appointment, scheduled_date: 0.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
+    create(:appointment, scheduled_date: 14.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
+    create(:appointment, scheduled_date: 15.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
+    create(:appointment, scheduled_date: 30.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
+    create(:appointment, scheduled_date: 31.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
+    create(:appointment, scheduled_date: 60.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
+    create(:appointment, scheduled_date: 61.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
+    create(:appointment, scheduled_date: 100.days.from_now, device_created_at: Time.current, facility: facility, patient: create(:patient, recorded_at: 1.month.ago))
```

Comment:
Want to put the dates in an array and loop instead? Will be more readable that way.